crates/turbopack/src/module_options/module_options_context.rs

```diff
@@ -159,6 +159,8 @@ pub struct ModuleOptionsContext {
     pub ignore_dynamic_requests: bool,
 
     pub use_swc_css: bool,
+
+    pub side_effect_free_packages: Vc<Vec<String>>,
```

Comment:
Should be use a `Vc<Glob>` instead? That would be more flexible.

Reply:
`the trait bound `Glob: turbo_tasks::Dynamic<Box<(dyn ValueDefault + 'static)>>` is not satisfied`

Implementing default for `Glob` does not look like a good idea.

Reply:
I used `Vec<String>` for config and `Glob` for actual operation.

Reply:
This could be `pub side_effect_free_packages: Option<Vc<Glob>>`, but we can refactor that in a follow-up